src/Plugin/GraphQL/DataProducer/Routing/RouteEntity.php

```diff
@@ -107,7 +107,15 @@ public function __construct(
    */
   public function resolve($url, ?string $language, FieldContext $context): ?Deferred {
     if ($url instanceof Url) {
+      // Check that this route is actually an entity route.
```

Comment:
Should we check `$url->isRouted()` before calling `$url->getRouteName()`

Reply:
Good idea, added the check